### PR TITLE
add-design

### PR DESCRIPTION
**Pull request by justynjozwiak**
designs implementation

### CODE REVIEW

File: tests/dummy/app/components/technology-entry.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+const { Component, get, computed } = Ember;
+
+export default Component.extend({
+  isSelected: computed('selectedTechnology', function() {
+    return get(this, 'technology.name') === get(this, 'selectedTechnology.name');
```
Comment (jbandura) [inline tests/dummy/app/components/technology-entry.js]: you could use `computed.equal` here

File: tests/dummy/app/routes/technology/socket-clients.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+const { Route, get } = Ember;
+
+export default Route.extend({
+  model(params, model) {
+    return {
+      routeName: get(this, 'routeName').split('.')[1].replace(/-/g, ' '),
```
Comment (jbandura) [inline tests/dummy/app/routes/technology/socket-clients.js]: I'd suggest extracting it to a mixin/helper function

File: tests/dummy/app/components/technology-entry.js (hunk 1 above)
Comment (justynjozwiak) [inline tests/dummy/app/components/technology-entry.js]: not working in any way :S

File: tests/dummy/app/routes/technology/socket-clients.js (hunk 2 above)
Comment (justynjozwiak) [inline tests/dummy/app/routes/technology/socket-clients.js]: will do this in PR with font change, I need to merge it and deploy ASAP

**Pull request merged**